index.html

```diff
@@ -278,47 +418,665 @@ <h4>Identification and Profile Signaling</h4>
         <p>TTML documents representing DAPT Scripts shall be compliant with the Text Profile defined in [[!ttml-imsc1.1]] with the additional constraints defined in this specification.</p>
         <p>TTML documents representing DAPT Scripts shall specify a <code>ttp:contentProfiles</code> attribute on the <code>tt</code> element with one value equal to <code>http://www.w3.org/ns/ttml/profile/dapt</code>. Other values such as the text profile designator [[!ttml-imsc1.1]] of may be present.</p>
       </section>
-      <section>
-        <h4>DAPT Namespace</h4>
-        <p>This specification defines the following namespaces:</p>
-        <table class='simple'>
+      <section id="profile-resolution-semantics">
+        <h3>Profile Resolution Semantics</h3>
+        <p>
+          For the purpose of content processing, the determination of the resolved profile SHOULD take into account both the signaled profile, as defined in <a href="#profile-signaling-section"></a>, and profile metadata, as designated by either (or both) the Document Interchange Context or (and) the Document Processing Context, which MAY entail inspecting document content.
+        </p>
+        <p>
+          If the resolved profile is not the Profile supported by the Processor but is feasibly interoperable with the Profile, then the resolved profile is the Profile.
+          If the resolved profile is undetermined or not supported by the Processor, then the Processor SHOULD nevertheless process the Document Instance using the Profile; otherwise, processing MAY be aborted.
+          If the resolved profile is not the proposed Profile, processing is outside the scope of this specification.
+        </p>
+        <p>
+          If the resolved profile is the profile supported by the Processor, then the Processor SHOULD process the Document Instance according to the Profile.
+        </p>
+      </section>
+    </section>
+    <section id="profile-constraints">
+      <h2>Constraints</h2>
+      <section id="Document Encoding">
+        <h3>Document Encoding</h3>
+        <p>
+          A Document Instance MUST use UTF-8 character encoding as specified in [[UNICODE]].
+        </p>
+      </section>
+      <section id="foreign-elements-and-attributes">
+        <h3>Foreign Element and Attributes</h3>
+        <p>
+          A <a>Document Instance</a> MAY contain elements and attributes that are neither specifically permitted nor forbidden by a profile.
```

Comment:
> by a profile
Can we rephrase and say by this specification?

Reply:
Yes if we are sure we're only defining one specification.

Reply:
We should open an issue to discuss whether we need one profile or two profiles (audio-support or not).

Reply:
https://github.com/w3c/dapt/issues/46